courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/3_keras_sequential_api.ipynb

```diff
@@ -156,6 +156,13 @@
     "We use a python dictionary comprehension to create the feature columns for our model, which is just an elegant alternative to a for loop."
```

Comment:
<p>As suggested below, have the order of learning objectives be sequential as they are through the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>